homeassistant/components/zwave_js/services.py

```diff
@@ -66,6 +66,14 @@ def broadcast_command(val: dict[str, Any]) -> dict[str, Any]:
     lambda value: int(value, 16),
 )
 
+VALUE_SCHEMA = vol.Any(
+    vol.Coerce(int),
```

Comment:
This will also convert a boolean to a integer. Is that ok? What is the following boolean validator then used for?

Reply:
Hmm probably not. The idea behind switching was to support converting the string version of true to true. Perhaps I can re-add bool at the beginning and keep cv.Boolean where it is? Do you think that's redundant?

Reply:
I don't think we need to add support for converting a string `"true"` to `True`.